plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableQuery.php

```diff
@@ -373,21 +373,23 @@ private function date_to_date_query_arg( $date ): array {
 	/**
 	 * Returns UTC-based date query arguments for a combination of local time dates and a date shorthand operator.
 	 *
-	 * @param  array $dates_raw Array of dates (in local time) to use in combination with the operator.
+	 * @param  array  $dates_raw Array of dates (in local time) to use in combination with the operator.
 	 * @param  string $operator One of the operators supported by date queries (<, <=, =, ..., >, >=).
 	 * @return array Partial date query arg with relevant dates now UTC-based.
 	 *
+	 * @throws \Exception If an invalid date shorthand operator is specified.
+	 * 
 	 * @since 8.2.0
 	 */
 	private function local_time_to_gmt_date_query( $dates_raw, $operator ) {
 		$result = array();
 
 		// Convert YYYY-MM-DD to UTC timestamp. Per https://github.com/woocommerce/woocommerce/wiki/wc_get_orders-and-WC_Order_Query#date only date is relevant (time is ignored).
 		foreach ( $dates_raw as &$raw_date ) {
-			$raw_date = is_numeric( $raw_date ) ? $raw_date : strtotime( get_gmt_from_date( date( 'Y-m-d', strtotime( $raw_date ) ) ) );
+			$raw_date = is_numeric( $raw_date ) ? $raw_date : strtotime( get_gmt_from_date( gmdate( 'Y-m-d', strtotime( $raw_date ) ) ) );
```

Comment:
same as above, let's leave this as `date`

Reply:
I have reverted the `gmdate()` to `date()` in PR.

Reply:
@alexflorisca Please check.